Task2.playground/Contents.swift

```diff
@@ -10,71 +10,88 @@
  func minimumElement() -> Int? // Returns the first minimum element in the stack. Important: the function should work in O(1).
 
  Use inheritance.
- */
 
-class Stack {
-    private var items = [Int]()
+ Use generics.
+ */
 
-    // Adds element on the top of the stack.
-    func push(element: Int) {
+class Stack<Element> {
+    private var items = [Element]()
+
+    /// Adds the element on the top of the stack.
+    ///
+    /// Use this method to add a single element on the top of the stack.
+    ///
+    ///     var someStack = [1, 2, 3]
+    ///     someStack.push(element: 10)
+    ///     print(someStack)
+    ///     // Prints "[1, 2, 3, 10]"
+    ///
+    /// - Parameter element: The element to add on the top of the stack.
```

Comment:
nit:
You already described that the element will be added on the top.
```suggestion
    /// - Parameter element: The element to add to the stack.
```